feature-account-api/src/main/java/jp/co/soramitsu/account/api/domain/model/MetaAccount.kt

```diff
@@ -97,3 +97,16 @@ fun MetaAccount.accountId(chain: Chain): ByteArray? {
         else -> substrateAccountId
     }
 }
+
+fun MetaAccount.userAddress(chain: Chain): String? {
```

Comment:
I think it will be better to convert accountId to address right in first when condition.